### DISCUSSION TITLE
Conditionals in FD

### DISCUSSION THREAD

**Discussion by atagg2** [brianguenter/FastDifferentiation.jl#95] [General]
Hi,  it was mentioned in the documentation that future work will contain support for passing derivatives through conditional statements.  I am just wondering if that is something that is being actively worked on, and will be available soon, or if it is not a priority for development right now.  Thank you!

**Comment by brianguenter** [comment]
It is a priority and I am currently working on it. In the latest version of
FD you can write expressions with conditionals. You cannot yet
differentiate through. That is coming next and is an active work item.

Having said that it will probably be 3-4 weeks, at least, before it is
done.
